.github/workflows/build.yml

```diff
@@ -1,85 +1,175 @@
-name: CI
+name: CI/CD
 
 on:
-  workflow_dispatch:
   push:
     branches:
       - master
   pull_request:
     branches:
       - master
+  workflow_dispatch:
+    inputs:
+      is-a-release:
+        description: Do you want to release? (Only works in the master branch)
+        type: boolean
+      operative-system:
+        type: choice
+        description: Chose the Operative System
+        options:
+          - ubuntu-latest
+          - ubuntu-20.04
+          - macos-latest
+          - macos-11
+          - windows-latest
+          - windows-2022
+        default: ubuntu-latest
+      java-version:
+        type: choice
+        description: Chose the Java version
+        options:
+          - 19
+          - 18
+          - 17
+          - 16
+          - 15
+          - 14
+          - 13
+          - 12
+          - 11
+          - 8
+        default: 17
 
-permissions:
-  checks: write
-  contents: write
-  pull-requests: write
+permissions: {}
 
 jobs:
-  build:
+  latest-java-version:
+    name: Get latest Java version
+    runs-on: ubuntu-latest
+    steps:
+      - name: API Adoptium - Get Java versions
+        run: |
+          LATEST_JAVA_VERSION=$(curl --silent \
+              -X 'GET' \
+              'https://api.adoptium.net/v3/assets/version/%5B1.0%2C111.0%5D?heap_size=normal&image_type=jdk&page=0&page_size=1&project=jdk&release_type=ga&sort_method=DEFAULT&sort_order=DESC&vendor=eclipse' \
```

Comment:
I think the `info/available_releases` endpoint would be a simpler call: `https://api.adoptium.net/v3/info/available_releases`

Then all you need to do is read the `most_recent_feature_release` field.

Reply:
Awesome! I applied the change in the commit [d551f0a](https://github.com/Col-E/InstrumentationServer/pull/11/commits/d551f0ab09928f1e5aa692bd17a54b65c7146c5d).